packages/ui/src/i18n/dictionaries/authenticator/pt.ts

```diff
@@ -0,0 +1,51 @@
+export const ptDict = {
+  'Back to Sign In': 'Voltar para Entrar',
+  'Change Password': 'Mudar senha',
+  Changing: 'Mudando',
+  Code: 'Código',
+  'Confirm Password': 'Confirme a Senha',
+  'Reset your Password': 'Redefina sua senha',
+  'Confirm Sign Up': 'Confirmar inscrição',
+  'Confirm SMS Code': 'Confirme o código SMS',
+  'Confirm TOTP Code': 'Confirme o código TOTP',
+  Confirm: 'confirme',
+  'Confirmation Code': 'Código de confirmação',
+  Confirming: 'Confirmando',
+  'Create Account': 'Criar Conta',
+  'Creating Account': 'Criando conta',
+  Email: 'O email',
+  'Enter your code': 'Insira seu código',
+  'Enter your username': 'entre com seu nome de usuário',
+  'Enter your phone number': 'Digite seu número de telefone',
+  'Enter your email': 'Digite seu e-mail',
+  'Forgot your password? ': 'Esqueceu sua senha? ',
+  'Hide password': 'Esconder a senha',
+  Loading: 'Carregando',
+  Username: 'Nome do usuário',
+  'New password': 'Nova Senha',
+  Password: 'Senha',
+  'Phone Number': 'Número de telefone',
+  'Resend Code': 'Reenviar código',
+  'Reset your password': 'Redefina sua senha',
+  'Send Code': 'Enviar código',
+  'Send code': 'Enviar código',
```

Comment:
Duplicate entries here:
```suggestion
  'Send Code': 'Enviar código',
```